### ISSUE TITLE
A11y Contrast Issue with Links

### ISSUE THREAD

**Issue by 12vanblart** [Microflash/jada#3]
**Describe the bug**
Currently `a` tags do not have a large enough contrast ratio to meet [WCAG AA requirements](https://webaim.org/resources/contrastchecker/). 

**To Reproduce**
I used [Accessibility Insights for the Web](https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni) and [WAVE](http://wave.webaim.org/) extensions for Chrome to check the contrast. 

**Expected behavior**
Links should have a large enough contrast that they are easy for all users to read. 

**Screenshots**
Current Ratio: 
![Current Contrast Ratio for links at 2.69](https://user-images.githubusercontent.com/16465776/71996361-131dc000-320a-11ea-8576-6b00568ed89c.png)

Example color to improve ratio:
![Contrast Checker with better ratio](https://user-images.githubusercontent.com/16465776/71996309-fc776900-3209-11ea-9ffe-41673ffe25d5.png)

**Comment by Microflash** [comment]
Thanks @12vanblart for pointing this out. This has been fixed in c53d3cf. Although Lighthouse is green for now, I'll work on creating a pipeline to do this check every time.
